src/components/views/messages/MessageActionBar.tsx

```diff
@@ -127,12 +129,14 @@ const ReactButton: React.FC<IReactButtonProps> = ({ mxEvent, reactions, onFocusC
 
 interface IMessageActionBarProps {
     mxEvent: MatrixEvent;
-    // The Relations model from the JS SDK for reactions to `mxEvent`
-    reactions?: any;  // TODO: types
+    reactions?: Relations;
+
+    getTile: () => any | null;
```

Comment:
The comment got lost on this type - please restore it.

also the types are in conflict with what is actually used: either this should be `unknown`, or the others should be `any`.